Buildings/Experimental/OpenBuildingControl/ASHRAE/G36/CompositeSequences/EconomizerMultiZone.mo

```diff
@@ -0,0 +1,185 @@
+within Buildings.Experimental.OpenBuildingControl.ASHRAE.G36.CompositeSequences;
+model EconomizerMultiZone "Multiple zone VAV AHU economizer control sequence"
+
+  parameter Boolean use_enthalpy = true
+    "Set to true if enthalpy measurement is used in addition to temperature measurement";
+  parameter Real kPIDamLim=1 "Gain of damper limit controller";
```

Comment:
This should be `kP`, not `kPI`. The gain is for the proportional part, not the integral part. Same for line 7.
Also, lines 8 and 9 should be TiDamLin, not PI